features/jsonapi/filtering.feature

```diff
@@ -23,6 +23,18 @@ Feature: JSON API filter handling
     And the JSON should be valid according to the JSON API schema
     And the JSON node "data" should have 0 elements
 
+  Scenario: Apply property filter based on the 'fields'
```

Comment:
I couldnt get the filters to work on my live example although they work in your tests, did you have any particular grouping in the entities or something to get this to work?

Reply:
Did you declare a [Property Filter](https://api-platform.com/docs/core/filters#property-filter) on your entity ?